kim_cnn/main.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+from __future__ import absolute_import
```

Comment:
__future__ module is used for compatibility of python2 and python3. Actually these statements are redundant.